server/config/local.json-dist

```diff
@@ -27,6 +27,11 @@
   "metrics": {
     "sample_rate": 1
   },
+  "sentry": {
+    "sample_rate": 1,
+    "endpoint": "https://sentry.prod.mozaws.net/api/31/store/",
+    "api_key": "8e7aaaa38fdb46e58ec0b3de61373492"
```

Comment:
FWIW, local.json *must* be updated for `npm run test-server` tests to pass. Can you add a [BREAKING CHANGE](https://docs.google.com/document/d/1QrDFcIiPjSLDn3EL15IJygNPiHORgU1_OOAqWjiDU5Y/edit#heading=h.gbbngquhe0qa) message to the bottom of the commit message?